pkg/templates/node_1.10.go

```diff
@@ -106,6 +113,9 @@ systemd:
       contents: |
         [Unit]
         Description=Kubelet via Hyperkube ACI
+        StartLimitAction=reboot
```

Comment:
I'm a little concerned that this will cause reboots down the road then the kubelet is crashing for other reasons. Maybe we should create a dedicated unit that only crashes when the current kernel parameters are not set correctly or the cgroup support is loaded in the wrong version.

Reply:
Good idea, but how to find out if it is loaded or not?

Reply:
Something like this should work:
```
    - name: legacy-cgroup-reboot.service
      enable: true
      contents: |
        [Unit]
        Description=Reboot if legacy cgroups are not enabled yet
        FailureAction=reboot

        [Service]
        Type=simple
        ExecStart=/usr/bin/grep 'cgroup' /proc/cmdline

        [Install]
        WantedBy=multi-user.target
```

Reply:
But I assume this will break current stable nodes.